pkg/config/templates/cos-after-install-chroot.yaml

```diff
@@ -0,0 +1,9 @@
+# yip stage to overwrite after-install-chroot layout
+if: '[ ! -f "/run/cos/recovery_mode" ]'
+name: "Run after-install-chroot commands"
+commands:
+    {{- if .OS.AfterInstallChrootCommands }}
+    {{- range $cmd := .OS.AfterInstallChrootCommands }}
+    - {{ $cmd }}
```

Comment:
Do we need to use `printf "%q"` here for better handling with the string `"`?
But I see the test code. It looks like we can already handle `"` well here.